doc/tutorials/content/narf_feature_extraction.rst

```diff
@@ -66,7 +66,7 @@ descriptors in the same place, but for different dominant rotations.
 The resulting PointCloud contains the type Narf36 (see
 common/include/pcl/point_types.h) and store the descriptor as a 36 elements
 float and x,y,z,roll,pitch,yaw to describe the local coordinate frame at which
-the feature was extracted. The descriptors can now be compared, e.g., whith the
+the feature was extracted. The descriptors can now be compared, e.g., with the
 Manhatten distance (sum of absolute differences).
```

Comment:
Since we're going through doc review, it should actually be Manhattan.

Reply:
Did you leave this requested change intentionally out?

Reply:
> Did you leave this requested change intentionally out?  
No, not intentionally. I didn't see it on this page for some reason. Will make the change now

Reply:
Done